### DISCUSSION TITLE
Reuniones (meetings)

### DISCUSSION THREAD

**Discussion by aleniso** [pmoros/the-graphus#11] [General]
### Bienvenidos a las reuniones del grupo.

**Comment by pmoros** [comment]
# Workshop meeting
## Details
- Date: 13/04/2021
- Start: 17:00
- End: 23:40
- Duration: 7 hours
- Topic: Adjustment of the requirements. Workshop use cases, Entity-Relationship diagrams.
- People: @aleniso @pmoros @Ferney021 

## Time usage

- Meeting preparation (10%): How should the missing team members help to improve what has been done
- Documents review (35%): Curricula and programs
- Topics discussion (30%): Workshop related topics, search and actual work
- Report preparation (5%):
- Assign tasks and responsibilities ():
- Others (20%): Unrelated topics.

**Comment by pmoros** [comment]
# Sync Meeting
## Details

    Date: 14/04/2021
    Start: 17:00
    End: 17:35
    Duration: 35 min
    Topic: Front end login and register check. 
    People: @pmoros @andgonzalezcas 

## Description

- What information goes to front after login
- The register and login will be displayed using a Modal
- Think all the information that will be displayed on the "Inscripción de asignaturas" view for each card
- Added member to the repository

## Time Usage

- Meeting preparation (): How should the missing team members help to improve what has been done
- Documents review (25%): Repository discussion and dates
- Topics discussion (30%):
- Report preparation (5%):
- Assign tasks and responsibilities (20%):
- Others (20%): Adding member to the repository

**Comment by aleniso** [comment]
# Sync Meeting
## Details
Date: 15/04/2021
Start: 17:00
End: 18:00
Duration: 60 min
Topic: ER model 
People: @aleniso @Ferney021 @ganzola 
## Description
- ER data base model
-  course requirements entity
## Time usage 
    Meeting preparation (5%): 
    Documents review (25%): 
    Topics discussion (30%):
    Report preparation ():
    Assign tasks and responsibilities (20%):
    Others (20%):

**Comment by aleniso** [comment]
# Workshop Meeting
## Details
Date: 15/04/2021
Start: 19:30
End: 22:30
Duration: 180 min
Topic: ER model 
People: @aleniso @pmoros  
## Description
- ER data base model
- course curricula 100%
- user management 90%
## Time usage 
    Meeting preparation (5%): 
    Documents review (35%): 
    Topics discussion (30%):
    Report preparation (15%):
    Assign tasks and responsibilities (5%):
    Others (10%):

**Comment by pmoros** [comment]
# Workshop Meeting
## Details
Date: 27/04/2021
Start: 20:25
End: 11:59
Duration: 4 hours
Topic: Sprint 2 - Planning
People: @aleniso @pmoros @ganzola 
## Description
- Define Goals
- Priority Criteria for user stories
- Pick User Stories for Sprint 2
- Add necessary User Stories
## Time usage 
    Meeting preparation (5%): 
    Documents review (35%): 
    Topics discussion (40%):
    Report preparation (5%):
    Assign tasks and responsibilities (10%):
    Others (5%):

**Comment by aleniso** [comment]
# Workshop Meeting
## Details
Date: 28/04/2021
Start: 4:00 pm
End:  10:00 pm
Duration: 
Topic: Sprint 2 - Planning
People: @aleniso @pmoros @ganzola 
## Description
- Define Goals
- Priority Criteria for user stories
- Pick User Stories for Sprint 2
- Add necessary User Stories
## Time usage 
    Meeting preparation (5%): 
    Documents review (35%): 
    Topics discussion (40%):
    Report preparation (5%):
    Assign tasks and responsibilities (10%):
    Others (5%):

**Comment by aleniso** [comment]
# Workshop Meeting
## Details
Date: 2/05/2021
Start: 8:30 pm
End: 10:15 pm
Duration: 
Topic: Sprint 3 - Planning
People: @aleniso @pmoros @ganzola  @Ferney021  @andgonzalezcas 
## Description
- Define Goals
- Priority Criteria for user stories
- Pick User Stories for Sprint 3
## Time usage 
    Meeting preparation (5%): 
    Documents review (10%): 
    Topics discussion (55%):
    Report preparation (5%):
    Assign tasks and responsibilities (20%):
    Others (5%):

**Comment by gabotachak** [comment]
# Workshop Meeting
## Details
Date: 3/05/2021
Start: 8:30 am
End: 9:00 am
Duration: 
Topic: Sprint 3
People: @aleniso @pmoros @ganzola  @Ferney021  @andgonzalezcas 
## Description
- Define Goals
- Priority Criteria for user stories
- Pick User Stories for Sprint 3
## Time usage 
    Meeting preparation (5%): 
    Documents review (10%): 
    Topics discussion (55%):
    Report preparation (5%):
    Assign tasks and responsibilities (20%):
    Others (5%):

**Comment by pmoros** [comment]
# Daily Scrum
## Details
Date: 12/05/2021
Start: 8:30 am
End: 9:00 am
Duration: 
Topic: Database Connection and management tools
People: @aleniso @pmoros @ganzola  @Ferney021  @andgonzalezcas 
## Description
- Progress tracked in Trello
- Heroku deployment
- Check database connection
## Time usage 
    Meeting preparation (5%): 
    Documents review (10%): 
    Topics discussion (55%):
    Report preparation (5%):
    Assign tasks and responsibilities (20%):
    Others (5%):

**Comment by gabotachak** [comment]
# Daily Scrum
## Details
Date: 19/05/2021
Start: 8:11 am
End: 9:00 am
Duration: 
Topic: Database Connection and management tools
People: @aleniso @pmoros @ganzola  @Ferney021  @andgonzalezcas
## Description
- Variables de entorno en React
- Interfaz de usuario
## Time usage 
    Meeting preparation (5%): 
    Documents review (10%): 
    Topics discussion (80%):
    Report preparation (5%):

**Comment by gabotachak** [comment]
# Daily Scrum
## Details
Date: 22/05/2021
Start: 11:50 am
End: 1:00 pm
Duration: 
Topic: Google Auth and entities CRUD
People: @pmoros @ganzola
## Description
- Verification of google auth token from frontend and using this on backend
- CRUD of different db entities
## Time usage 
    Meeting preparation (5%): 
    Google Auth (40%):
    Entities (40%):
    Endpoints (15%):

**Comment by gabotachak** [comment]
# Login meeting
## Details
Date: 22/05/2021
Start: 4:00 pm
End: 4:40 pm
Duration: 40 min
Topic: Google Auth between frontend and backend
People: @andgonzalezcas  @ganzola
## Description
- definition of endpoints between frontend and backend for login and setting this endpoint for backend: [POST] /login with this payload:
```json
{
    "tokenId": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjQ4NmYxNjQ4MjAwNWEyY2RhZjI2ZDkyMTQwMThkMDI5Y2E0NmZiNTYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjc2MTgzMTE4ODIzLXJoYmlwaGVwYTliczNic2kwNG1saXFuNWluYnA1bmxlLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjc2MTgzMTE4ODIzLXJoYmlwaGVwYTliczNic2kwNG1saXFuNWluYnA1bmxlLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTA5MzExMzU0MjQyNjk4ODU2MDE2IiwiaGQiOiJ1bmFsLmVkdS5jbyIsImVtYWlsIjoiZ2Fuem9sYUB1bmFsLmVkdS5jbyIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJhdF9oYXNoIjoidGEtM0RzNUFBMnpuODQ5aEJUZFNEdyIsIm5hbWUiOiJHYWJyaWVsIEFuZHJlcyBBbnpvbGEgVGFjaGFrIiwicGljdHVyZSI6Imh0dHBzOi8vbGgzLmdvb2dsZXVzZXJjb250ZW50LmNvbS9hLS9BT2gxNEdnYWZFUDR6bzJMQ29yOEkyLWprSHI0Wk9YbHFwWm5kMl9PdnFCSHNRPXM5Ni1jIiwiZ2l2ZW5fbmFtZSI6IkdhYnJpZWwgQW5kcmVzIiwiZmFtaWx5X25hbWUiOiJBbnpvbGEgVGFjaGFrIiwibG9jYWxlIjoiZW4tR0IiLCJpYXQiOjE2NTMyNTQxNjEsImV4cCI6MTY1MzI1Nzc2MSwianRpIjoiMjIwNjgyYjYzYjIyYjgxZGMzNmZiNDczZWYxZWYzNjZjMDYxNjI0MiJ9.THRsGGe7fBpgVIpLmQuj3IhN2l0VQ1UaemGa_fVlXK6YQ25Oi0Eb6JaaKMgolQfy-5cvjtekVghoniCD23cDJpSQGbHtywqsaYR-IrLZ40HV999m9PRdhkKk1GtWH1CgrjdOMGXys8vGMedtWXcBfv-sZ_llqsNsNzeOIJKUN7zWZmv8hqYvuuoGoSMZJThW11aixVXFAGU6BzMRO0qnovGk8fp7boJusWxPbWaah6_Ss-QhW8yuZ0gAjBRi-w8h2ZGz-a_pz80vdR1-ZFcpiK9zZvqiuC83BIvHtf2FgKamNw20XIIQeZXQWIygr4xfqVLy9eGF49LFemGnWXWYAg"
}
```

## Time usage 
    Discussion (100%):
